app/views/posts/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<div id="posts">
+<% @posts.each do |post| %>
+  <h2 id="post-title"><%= link_to post.title, post %></h2>
+  <p id="post-body"><%= post.body %> </p>
```

Comment:
We only use `id`'s on a single element. While an `id` is technically reusable, that's what we use classes for.